pages/doc/sources_managing.md

```diff
@@ -6,18 +6,34 @@ sidebar: doc_sidebar
 permalink: sources_managing.html
 summary: Learn about sources and how to manage them.
 ---
-In VMware Aria Operations for Applications (formerly known as Tanzu Observability by Wavefront), a **source** is a unique application, host, container, or instance that emits metrics. The source is explicitly set in the `source` field of an [Operations for Applications data format](wavefront_data_format.html) metric. For AWS integrations, the source is extracted from [AWS Cloud Watch service properties or dimensions](integrations_aws_metrics.html#aws_sources).
 
-* To view and manage sources, select **Browse > Sources**.
+In VMware Aria Operations for Applications (formerly known as Tanzu Observability by Wavefront), a **source** is a unique application, host, container, or instance that emits metrics. The source is explicitly set in the `source` field of an [Operations for Applications data format](wavefront_data_format.html) metric. For AWS integrations, the source is extracted from the [AWS CloudWatch service properties or dimensions](integrations_aws_metrics.html#aws_sources).
+
+* To view and manage sources, click **Browse > Sources** on the toolbar.
 * To filter and group in charts, use the Wavefront Query Language. You can filter by source and [filter and group by source tags](tags_overview.html#source-tags).
 
 {% include shared/permissions.html entity="sources" entitymgmt="Source Tag" %}
 
 ## Examine Sources in the Source Browser
 
-If you don't see metrics in charts that filter by source, you can examine sources in the Source browser.
+If you don't see metrics in charts that filter by source, you can examine sources in the Sources browser.
+
+{% include note.html content="The Sources browser doesn't filter out the obsolete sources." %}  
+
+![An annotated screenshot of the sources browser, where what you can do is listed in the bullet list bellow.](/images/sources-browser-page.png)
+
+On the Sources browser, you can:
 
-{% include note.html content="The Sources browser doesn't filter out the obsolete sources." %}
+* Search for and apply filters to the sources so that you can narrow down the results.
```

Comment:
Search for sources and apply filters to the sources so that you can narrow down the results.